Source/WebCore/rendering/RenderBlockFlow.cpp

```diff
@@ -3049,7 +3049,7 @@ std::optional<LayoutUnit> RenderBlockFlow::inlineBlockBaseline(LineDirectionMode
             return std::nullopt;
     }
     // Note that here we only take the left and bottom into consideration. Our caller takes the right and top into consideration.
-    float boxHeight = synthesizedBaselineFromBorderBox(*this, lineDirection) + (lineDirection == HorizontalLine ? m_marginBox.bottom() : m_marginBox.left());
+    float boxHeight = synthesizedBaseline(*this, parentStyle(), lineDirection, Border) + (lineDirection == HorizontalLine ? m_marginBox.bottom() : m_marginBox.left());
```

Comment:
any particular reason for this to be float?

Reply:
Not as far as I can tell. Changed this to auto and changed `lastBaseline` below it to LayoutUnit